ui/app/document-upload/controllers/documentController.js

```diff
@@ -142,9 +143,21 @@ angular.module('opd.documentupload')
                         v: "custom:(uuid,setMembers:(uuid,name:(name)))"
                     }
                 }).then(function (response) {
+                    if (response.data.results[0].setMembers && response.data.results[0].setMembers.length > 0) {
```

Comment:
Please ensure do empty check. If there is no TopLevelConcept then here the application will enter into exception state.
TypeError: Cannot read property 'setMembers' of undefined
    at https://192.168.33.10/bahmni/document-upload/controllers/documentController.js:146:50